src/model/Project.ts

```diff
@@ -211,6 +214,61 @@ export default class Project extends Ressource {
     return Domain.query({ projectId: this.id, limit }, this.getLink("domains"));
   }
 
+  /**
+   * Get a list of environment domains for the project.
+   *
+   * @param int limit
+   *
+   * @return EnvironmentDomain[]
+   */
+  async getEnvironmentDomains(environmentId: string) {
+    return EnvironmentDomain.query(
+      { projectId: this.id, environmentId },
+      this.getLink(`environments/${environmentId}/domains`)
+    );
+  }
+
+  /**
+   * Add a environmentDomain to the environment project.
+   *
+   * @param string name
+   * @param string environmentId
+   * @param string type
+   * @param string replacement_for
+   * @param array  ssl
+   *
+   * @return Result
+   */
+  async addEnvironmentDomain(
+    name: string,
+    environmentId: string,
+    type = "replacement",
+    replacement_for?: string,
+    ssl = []
```

Comment:
It should be cleaner to have:
```suggestion
    ssl?: Array<string>
```

After you can have:
```js
    if (ssl) {
      body.ssl = ssl;
    }
```

Reply:
Unfortunately, If I do like this there is a Typescript error in here